operations/addition.py

```diff
@@ -0,0 +1,4 @@
+# Addition by two numbers
+def addition(num_1, num_2):
+    result = num_1 + num_2
```

Comment:
Unnecessary intermediate variable `result`.
```suggestion
    return num_1 + num_2
```